.markdown-doctest-setup.js

```diff
@@ -0,0 +1,53 @@
+function noop () {}
+
+function jQuery () {
+  return {
+    on: function (ev, cb) {
+      cb()
+    }
+  }
+}
+
```

Comment:
jQuery is used 12 times in the examples. We could add it to the devDependencies so we don't have to shim it, but this solution turned out to work quite well. Using the real jQuery would also mean we would need a browser context, or to shim out all the parts of document that jQuery uses.